### PR TITLE
Add command to delete other islands

### PR DESCRIPTION
**Pull request by dlsf**
I don't really like this solution, but I guess it's alright for now

### CODE REVIEW

**Comment by dlsf** [comment]
This is not ready yet

File: src/main/java/com/iridium/iridiumskyblock/commands/DeleteCommand.java [hunk 1]
```diff
@@ -54,6 +67,17 @@ public boolean execute(CommandSender sender, String[] args) {
         return false;
     }
 
+    private void deleteOtherIsland(CommandSender commandSender, String playerName) {
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/commands/DeleteCommand.java:70]: not sure if I like this being a seperate method

Comment (dlsf) [reply src/main/java/com/iridium/iridiumskyblock/commands/DeleteCommand.java:70]: Sure, I can inline it, but imo it's much more readable this way

Comment (dlsf) [reply src/main/java/com/iridium/iridiumskyblock/commands/DeleteCommand.java:70]: It would look like this: https://pasty.lus.pm/UuHIyP

**Pull request merged**